packages/devtools_app/test/scenes/memory/diff_snapshot.dart

```diff
@@ -58,9 +55,7 @@ class DiffSnapshotScene extends Scene {
   @override
   String get title => '$DiffSnapshotScene';
 
-  void tearDown() {
-    FeatureFlags.memoryDiffing = false;
-  }
+  void tearDown() {}
```

Comment:
This isn't needed anymore.

Reply:
I want it to be invoked by consumers in case we will want to put something here.